x-pack/plugins/ml/public/file_datavisualizer/components/results_links/results_links.js

```diff
@@ -20,6 +20,8 @@ import moment from 'moment';
 import uiChrome from 'ui/chrome';
 import { ml } from '../../../services/ml_api_service';
 
+const RECHECK_DELAY = 3000;
```

Comment:
Is this a delay in ms? We often add `_MS` to these constants to make it explicit.